seqlike/SeqLike.py

```diff
@@ -187,14 +187,12 @@ def aa(self, auto_translate=True, **kwargs) -> "SeqLike":
         :param auto_translate: Whether to automagically translate
```

Comment:
Above this line, we should explain why the default behaviour of `.aa()` preserves all of the NT record attributes.

My strawman:

```
We preserve all NT record attributes
because it becomes convenient for sequence manipulation tasks
where the starter sequence is an NT and we merely want the AA form.
The default behaviour, thus, has `id=True`, `name=True`, `description=True`,
`annotations=True`, and `dbxrefs=True`.
To change the behaviour, you can set any of those to False in `.aa()`, for example:

<put the Markdown triple back-ticks here>
    seq.aa(id=False)
<end triple back-ticks>
```